addons/product/security/product_security.xml

```diff
@@ -13,6 +13,14 @@
         <field name="implied_ids" eval="[(4, ref('product.group_product_pricelist'))]"/>
     </record>
 
+    <record id="base.group_multi_currency" model="res.groups">
+        <!--
+            Sales & POS multi-currency is based on pricelists, if multi-currency is enabled,
+            enable pricelists.
+        -->
+        <field name="implied_ids" eval="[(4, ref('product.group_product_pricelist'))]"/>
+    </record>
```

Comment:
I guess this can be removed then

Reply:
Indeed. This is already managed in `_activate_group_multi_currency`

Reply:
This enables the pricelist automatically if the multi-currency is enabled, without having to reprocess the settings (and without a post-init hook).
And yes the comment is useful since we already received the question why there was such a relation between currency and pricelists group.

Reply:
@Feyensv  doesn't the override of `_activate_group_multi_currency` achieve this? and it has comment too

Reply:
* install accounting
* enable other currencies
* install sale
* Without the implied_group, pricelists won't be enabled automatically.

Reply:
Not true, due to the following override
`group_user._apply_group(self.env.ref('product.group_product_pricelist'))`

So I have tested the following (after removing the XML implied group):

- Install invoicing
- enable a second currency (`group_multi_currency` and `group_product_pricelist` are now implied in `group_user`)
- disable second currency (`group_multi_currency` is removed, but `group_product_pricelist` is still implied)
- install sale
- Add a new user, basic pricelist is active.

The moment group_multi_currency is activated, group_product_pricelist becomes an implied group of the internal user group - and can only be removed manually

Reply:
My bad, I forgot you moved this from sale to product :man_facepalming:, which is installed before account :).
My remark would still be true if you modify currencies with only base installed, before installing product, but that's an extreme case not worth covering.

Reply:
That's only possible via the api or shell. And since product is a
dependency for account, and by implication sale and point_of_sale, we
should be safe.